config/yugabyte/regression_pipelines/join_workloads/postgres/JOING12_join_scale_parallel_workers.yaml

```diff
@@ -0,0 +1,104 @@
+                      util: RandomNoWithDecimalPoints
+                      params: [ 1, 1000000, 2 ]
+                    - name: col_float5_
+                      count: 2
+                      util: RandomNoWithDecimalPoints
+                      params: [ 1, 1000000, 5 ]
+                    - name: col_boolean_
+                      count: 1
+                      util: RandomBoolean
+                    - name: col_varchar10_id_
+                      count: 1
+                      util: PrimaryStringGen
+                      params: [1, 10]
+                    - name: col_varchar100_id_
+                      count: 2
+                      util: PrimaryStringGen
+                      params: [1, 100]
+                    - name: col_varchar500_id_
+                      count: 1
+                      util: PrimaryStringGen
+                      params: [1, 500]
+
+        executeRules:
+            -   workload: JOING12_1_join_with_2_parallel_workers_project_aggregate
+                customTags: schematype=regular,partition=hash,pkey=hash,cardinality=multiple,projectition=aggregate,queryshape=join,joinon=pkey
+                run:
+                    -   name: join_on_pk_3tbl
+                        weight: 100
+                        queries:
+                            -   query: /*+ set(max_parallel_workers_per_gather 2) */ select count(*) from pkeyBigint1M_1 as a join pkeyBigint1M_2 as b on a.col_bigint_card1_1=b.col_bigint_card1_1 and a.col_bigint_card3_1=b.col_bigint_card3_1 where b.col_bigint_card1_1>1000000 and  b.col_bigint_card2_1 >1000000;
+
+
+            -   workload: JOING12_2_join_with_4_parallel_workers_project_aggregate
+                customTags: schematype=regular,partition=hash,pkey=hash,cardinality=multiple,projectition=aggregate,queryshape=join,joinon=pkey
+                run:
+                    -   name: join_on_pk_3tbl
+                        weight: 100
+                        queries:
+                            -   query: /*+ set(max_parallel_workers_per_gather 4) */ select count(*) from pkeyBigint1M_1 as a join pkeyBigint1M_2 as b on a.col_bigint_card1_1=b.col_bigint_card1_1 and a.col_bigint_card3_1=b.col_bigint_card3_1 where b.col_bigint_card1_1>1000000 and  b.col_bigint_card2_1 >1000000;
+
+            -   workload: JOING12_3_join_with_8_parallel_workers_project_aggregate
+                customTags: schematype=regular,partition=hash,pkey=hash,cardinality=multiple,projectition=aggregate,queryshape=join,joinon=pkey
+                run:
+                    -   name: join_on_pk_3tbl
+                        weight: 100
+                        queries:
+                            -   query: /*+ set(max_parallel_workers_per_gather 8) */ select count(*) from pkeyBigint1M_1 as a join pkeyBigint1M_2 as b on a.col_bigint_card1_1=b.col_bigint_card1_1 and a.col_bigint_card3_1=b.col_bigint_card3_1 where b.col_bigint_card1_1>1000000 and  b.col_bigint_card2_1 >1000000;
```

Comment:
Right now the query is running for 26 seconds. Do we want to reduce the time of the query to say 2-3 seconds. This will ensure that the query is run multiple times. Right now, the query is being run only 4-10 times